prediction_market_agent/agents/safe_guard_agent/app.py

```diff
@@ -10,8 +10,7 @@
 
 nest_asyncio.apply()  # Required for streamlit to work with asyncio.
 st.set_page_config(layout="wide")
-add_sink_to_logger()
-
+add_sink_to_logger(explicit=True)
```

Comment:
If explicit=True is used, then only logs with the extra attribute `streamlit=True` will be printed in Streamlit app. Eg:

```
logger.info("This is not visible on stremalit")
logger.info("But this is", streamlit=True)
```

It's handy if you want fine-grained control over the streamlit app, but don't want to get rid of handy logs for production.